File: PaMVPDemo/lib-network/src/main/java/com/pa/network/bean/ResultException.java
package com.pa.network.bean;

/**
 * Created by wen on 2018/5/14.
 */
public class ResultException extends RuntimeException {

    private String errCode = "";

    public ResultException(String errCode, String msg) {
        super(msg);
        this.errCode = errCode;
    }

    public String getErrCode() {
        return errCode;
    }
}

File: PaMVPKotlinDemo/lib-network/src/main/java/com/pa/network/bean/ResultException.kt
package com.pa.network.bean

/**
 * Created by wen on 2018/5/14.
 */
class ResultException(errCode: String?, msg: String?) : RuntimeException(msg) {

    var errCode : String? = ""

    init {
        this.errCode = errCode
    }
}
